scripts/test_arduino_library.sh

```diff
@@ -55,13 +53,20 @@ InstallLibraryDependencies () {
   wget -O /tmp/arducam-master.zip https://github.com/ArduCAM/Arduino/archive/e216049ba304048ec9bb29adfc2cc24c16f589b1/master.zip
   unzip -o /tmp/arducam-master.zip -d /tmp
   cp -a /tmp/Arduino-e216049ba304048ec9bb29adfc2cc24c16f589b1/ArduCAM "${ARDUINO_LIBRARIES_DIR}"
+
+  sed --in-place '/#define DEBUG_SERIAL_OBJECT/s/(Serial)/(Serial1)/' src/tensorflow/lite/micro/system_setup.cpp
```

Comment:
I'm not sure if this is a proper place to do it. We don't want to affect regular users. This should be at least discussed with tflite team

Reply:
I'd prefer a separate stage in the CI workflow, before testing in Renode, that does this.

Reply:
I moved it to a separate step for now then